contrib/win32/install/install/product.wxs

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?if $(var.MANUFACTURER) = "" ?>
+	<?define MANUFACTURER = "Microsoft Corporation"?>
+<?endif?>
+
+<?if $(var.SOURCEDIR) = "" ?>
+  <?error Variable SOURCEDIR not set?>
+<?endif?>
+
+<?if $(var.VERSION) = "" ?>
+  <?define VERSION = "0.0.0.0"?>
+<?endif?>
+
+<?if $(sys.BUILDARCH)="x86" ?>
+  <?define PROGRAMFILESFOLDER="ProgramFilesFolder"?>
+<?elseif $(sys.BUILDARCH)="x64" ?>
+  <?define PROGRAMFILESFOLDER="ProgramFiles64Folder"?>
+<?else?>
+  <?error Unsupported value of sys.BUILDARCH=$(sys.BUILDARCH)?>
+<?endif?>
+
+<?define UPGRADECODERELEASE = "c7a00fb1-c477-40d3-a951-ec4c749da439" ?>
```

Comment:
These need to vary by bitness as well, which you'll need to pass in.

Reply:
Exactly... but this is only relevant if we really have to support 32bit with OpenSSH. Right?

Reply:
Yes, but are you suggesting that 32-bit Windows doesn't have OpenSSH? I honestly don't know - I haven't installed a 32-bit OS in over a decade.

Reply:
See that's my point as well. Never used 32-bit OS for years now. Perhaps we could say goodbye to this architecture here. @SteveL-MSFT what's you opinion on this? Do we have to support 32bit or not?

Reply:
Introduced upgrade codes for x86 and x64